From: Alexander Larsson <alexl@redhat.com>
Date: Mon, 28 Nov 2016 13:58:19 +0100
Subject: pull: scan_commit_object() - don't load variant twice

ostree_repo_load_commit already loaded the object, no need
to load it twice.

Closes: #595
Approved by: cgwalters
Origin: upstream, 2016.15, commit:c9b158e
---
 src/libostree/ostree-repo-pull.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/libostree/ostree-repo-pull.c b/src/libostree/ostree-repo-pull.c
index 9e96079..6f32c77 100644
--- a/src/libostree/ostree-repo-pull.c
+++ b/src/libostree/ostree-repo-pull.c
@@ -1137,10 +1137,6 @@ scan_commit_object (OtPullData         *pull_data,
   is_partial = pull_data->legacy_transaction_resuming
     || (commitstate & OSTREE_REPO_COMMIT_STATE_PARTIAL) > 0;
 
-  if (!ostree_repo_load_variant (pull_data->repo, OSTREE_OBJECT_TYPE_COMMIT, checksum,
-                                 &commit, error))
-    goto out;
-
   /* PARSE OSTREE_SERIALIZED_COMMIT_VARIANT */
   g_variant_get_child (commit, 1, "@ay", &parent_csum);
   if (g_variant_n_children (parent_csum) > 0)
